setup.py

```diff
@@ -68,6 +68,6 @@
             'check-python-versions = check_python_versions.cli:main',
         ],
     },
-    install_requires=['pyyaml'],
+    install_requires=['pyyaml', 'pytomlpp'],
```

Comment:
I'd rather use `tomli; python_version < 3.11` (except with the correct syntax for the python version check).

Reply:
Why are Appveyor builds failing to build pytomlpp?  I see Windows wheels on PyPI, why is pip install not using them?

Reply:
It's not clear for me why pytomlpp is building on windows even if wheel exists, I've never used appveyor so I've no idea of what's happening.

I've no problems to change TOML package, even if I saw pytomlpp is faster, I use it just for this reason.

Reply:
[tomlkit](https://github.com/sdispater/tomlkit) seems to be the only library that preserves comments.

[tomli](https://pypi.org/project/tomli/) has an explicit FAQ answer about comments.